### DISCUSSION TITLE
Render Video inside Model viewer without model

### DISCUSSION THREAD

**Discussion by Akhil-coditas** [google/model-viewer#4295] [Q&A]
https://github.com/google/model-viewer/assets/106797543/8c00393e-cb17-4a28-8eac-a4e53d9e9211
requirment -
on clicking activate button i need to show door video... and then my custom video .
in between door video where you see the green part , camera must be shown with the custom video in ar
and my door video should take the whole screen size so that it looks like door is opening and my custom video is being played inside it on ar . hope you got the requirement.

i will use webm video later though as you suggested but for now

what im doing is im loading door video first on my model...then im replacing door video with the custom video.

so how do i load door video which will take entire screen size..and then when the door opening video is completed..my custom video must be played in center in ar

**Comment by danilokrulj** [comment]
I'm not sure but I think to cover the entire screen you will need a div that covers the whole page, and even if that worked it would only work on WebXR (Android). And I'm not sure about the greenscreen but I used a chromakey extension to get that working in AR but I was using the MindAR library, not sure if that would work with model-viewer, but try looking into that.
